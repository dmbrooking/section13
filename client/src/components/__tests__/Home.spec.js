import React from 'react'

import Home from '../Home'

describe('<Home />', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Home />)
  })

  it('matches saved snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
