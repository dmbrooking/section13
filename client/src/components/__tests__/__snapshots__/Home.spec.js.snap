// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> matches saved snapshot 1`] = `
<div>
  Home
</div>
`;
